### ISSUE TITLE
Clarify Germline Analysis Pipeline in User Guide

### ISSUE THREAD

**Issue by DarioS** [etal/cnvkit#420]
I am interested in calculating copy number values for each sample in set of normal (blood) samples. I read the Germline Analysis section, which recommended the call command. But, the call module requires CNS files. Could the Germline Analysis section contain an example similar to the example showing the batch command for tumour-normal analysis in the pipeline section? It'd be convenient to use batch for germline analysis also, although I'm unsure if it's possible.

**Comment by etal** [comment]
Sure, I can add that. The `batch` example is essentially the same either way -- you provide case and control BAM files, and the pipeline constructs a pooled reference from the control BAMs, normalizes each case BAM to that reference, and performs segmentation on the normalized, binned coverage depth log-ratios. The output of segmentation is a `.cns` file for each case BAM file, and the `batch` command will produce these.

**Comment by DarioS** [comment]
You seem to describe copy number calculation of the cancer samples, but I was searching for an example of germline copy number analysis (i.e. absolute copy number estimates for each of the normal samples of a tumour-normal sample set, assuming that most regions of the genome have two copies/chromosomes).

**Comment by dantaki** [comment]
Does this mean CNVkit cannot call CNVs without a "control" panel? What if I want to make germline calls on a 30X whole genome for one sample at a time?

**Comment by FriederikeHanssen** [comment]
Bumping this question: can I use batch on the normal, unpaired, WGS samples if I create a flat reference?

**Comment by etal** [comment]
Yes, you can, but I'd recommend interpreting the results cautiously. The results on WGS without a robust reference built from a pool of control samples tend to be too noisy to reliably call most germline alterations, which are usually small, but it could maybe catch a larger-scale alteration if you have reason to believe that could be present in your sample. For more reliable results I'd recommend a method specifically designed for WGS like Parliament2 or Manta.
